certbot/log.py

```diff
@@ -299,7 +300,23 @@ def except_hook(exc_type, exc_value, trace, debug, log_path):
             _, _, exc_str = str(exc_value).partition(':: ')
             print(exc_str, file=sys.stderr)
         else:
-            traceback.print_exception(exc_type, exc_value, None)
+            """ Regex for parsing error messages"""
+            """ Regular expression groups"""
+            """ Match Group #1: Attempted connection host"""
+            """ Match Group #2: urllib3 Error Number"""
+            """ Match Group #3: urllib3 Error String"""
+            """ Example:"""
```

Comment:
For the example, I was thinking a full example error message before parsing.

Also, triple quotes (`"""`) are generally used for multi-line comments in Python, so open with a triple quote at the top, then close with another triple quote at the bottom, rather than repeating the quotes on each line.